templates/typescript_gapic/_util.njk

```diff
@@ -163,6 +181,11 @@ limitations under the License.
  *   An object stream which emits an object representing {{ typeLink(method.pagingResponseType) }} on 'data' event.
 {%- endmacro -%}
 
+{%- macro printReturnPageAsync(method) %}
+ * @returns {Object}
+ *   An iterable Object whose iterable properties are to be iterated over.
```

Comment:
This phase is cool but does not explain much to a person who has no idea :) 

Maybe write something like `An iterable Object that conforms to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols`?